public/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
Is it possible to create this into an individual component? Whereas in future there will be many implementations in this part.

Reply:
Yes, it's possible, but since we don't plan to dynamicly change `<meta>` , we can use the generated one as our “entry point”.

Reply:
Okay then, lets leave it first. Will be implement in future.